web_src/less/_base.less

```diff
@@ -1535,8 +1643,10 @@ a.ui.label:hover {
   }
 }
 
-.octicon-tiny {
-  font-size: .85714286rem;
+.octicon-tiny > .svg {
+  margin-top: -2px;
+  margin-bottom: -2px;
+  vertical-align: -2px;
 }
```

Comment:
This is only used in two places, suggest to remove the class and use helper classes if possible. Also, the previous `font-sizes` does nothing on SVG.